src/xrc/xh_bttn.cpp

```diff
@@ -47,7 +47,12 @@ wxObject *wxButtonXmlHandler::DoCreateResource()
     if (GetBool(wxT("default"), 0))
         button->SetDefault();
 
-    if ( GetParamNode("bitmap") )
+    if ( HasParam("bitmaps") )
+    {
+        button->SetBitmap(GetBitmapBundle("bitmaps", wxART_BUTTON),
+                          GetDirection("bitmapposition"));
+    }
+    else if ( GetParamNode("bitmap") )
```

Comment:
I don't know if it's really worth changing it now, but I wonder if it could have been better to add `GetBitmapOrBitmaps(paramBitmapName, paramBitmapsName, ...)` and check for both the bitmaps and the bitmap inside this function? It looks like we could then do this check only once, inside it, rather than in all of its callers, which would seem to be preferable.